### ISSUE TITLE
NPM Package instructions, usage with webPack?

### ISSUE THREAD

**Issue by davidwieler** [Voog/wysihtml#374]
Any way to get some instructions on how to require this with webpack in the docs?

I spent a few hours trying to figure it out, and in the end just ended up deleting the module.

**Comment by dmr** [comment]
Did you solve the problem? I have the same issue here

**Comment by davidwieler** [comment]
@dmr Not at all. Due to the lack of support, and my ticket being closed with no resolution I removed wysihtml and went with Summernote. It's more customizable and just works.

**Comment by dmr** [comment]
Thank you for your reply @davidwieler !
I guess we'll go the same path.
I'm just looking for a reliable "paste from word" Editor...

**Comment by davidwieler** [comment]
TinyMCE is a really good option and has that. We set up the newest version a couple weeks ago and it was a breeze.

**Comment by davidwieler** [comment]
@dmr ^^

**Comment by dmr** [comment]
Thank you for your reply. I'll try it

**Comment by pitpit** [comment]
If someone need it, you can enable wysihtml using https://webpack.js.org/loaders/script-loader/  that way:

```
import 'script-loader!wysihtml/dist/wysihtml.js';
import 'script-loader!wysihtml/dist/wysihtml.toolbar.js';
````

Unfortunately I did not manage to make it work w/o script-loader
